integration-tests/src/main/java/io/druid/testing/ConfigFileConfigProvider.java

```diff
@@ -53,69 +55,95 @@ private void loadProperties(String configFile)
     ObjectMapper jsonMapper = new ObjectMapper();
     try {
       props = jsonMapper.readValue(
-          new File(configFile), new TypeReference<Map<String, String>>()
-      {
-      }
+        new File(configFile), new TypeReference<Map<String, String>>()
+        {
+        }
       );
     }
     catch (IOException ex) {
       throw new RuntimeException(ex);
     }
     // there might not be a router; we want routerHost to be null in that case
-    routerHost = props.get("router_host");
-    if (null != routerHost) {
-	routerHost += ":" + props.get("router_port");
+    routerUrl = props.get("router_url");
+    if (routerUrl == null) {
```

Comment:
We are already logging the final URLs being used at the end of file. 
So, skipped adding a redundant log for every config here, 
will add logging if you feel strongly about it.